docs/extend/create_custom_operator.rst

```diff
@@ -0,0 +1,124 @@
+Create a custom operator
```

Comment:
Can we make this file look good on the github as well?

Reply:
@JanuszL do you have any idea how to do that?

Reply:
Currently no, as I'm not that familiar with what GH supports regarding rst. Please do some reading and check our options.

Reply:
@JanuszL Solved by making a python notebook